docs/test-fairy/using-test-fairy/uploading-an-app.md

```diff
@@ -8,5 +8,46 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-/Getting_Started/Upload_Apps.html
-docs/010_Getting_Started/300_Upload_Apps.md
\ No newline at end of file
+## Uploading an Application
+
+Once your account is created and verified, you are ready to upload your app.
+
+To upload an app, in the top menu, click **New Upload**.
+
+We recommend using the Upload API method to enable the Jenkins plugin, Gradle plugin, or Command line uploader.
+
+The code of our command line uploader, Jenkins plugin, and Gradle plugin is open source, so feel free to change and improve it.
+
+## Manual Upload
+
+### Supported Plaforms
+
+- **Android**: TestFairy supports uploading and distributing Android Applications. In order to distribute Android apps with TestFairy, they must be packaged as an `.apk` or `.aab` file.
+- **iOS**: TestFairy supports uploading and distributing iOS applications. iOS apps can be signed with AdHoc, Development, or Enterprise certificates. To distribute iOS apps with TestFairy, they must be packaged as an `.ipa` file.
+- **MacOS**: MacOS apps are bundled as `.app` files, however, to distribute MacOS apps with TestFairy, those `.app` files must be zipped into a `.zip` file.
+
+### Choose your Build File
+
+In the first stage you need to choose the file you want to upload. It can be an **.ipa** (for iOS), **.apk** (for Android), or **.zip** (for MacOS) file.
```

Comment:
```suggestion
In the first stage, choose the file you want to upload. It can be a `.ipa` (for iOS), `.apk` (for Android), or `.zip` (for MacOS) file.
```